### PR TITLE
Update transitions.md

### PR DESCRIPTION
**Pull request by jhserodio**
change transition exemple

### CODE REVIEW

File: src/guide/transitions.md [hunk 1]
```diff
@@ -35,7 +35,10 @@ When an element with transition is inserted or removed, Vue will:
 A typical CSS transition looks like this:
 
 ``` html
-<div v-if="show" transition="expand">hello</div>
+<div id="app">
+    <div v-if="show" transition="expand">hello</div>
+    <button @click="toggle">botao</button>
```
Comment (yyx990803) [inline src/guide/transitions.md]: What does botao mean?

Comment (jhserodio) [inline src/guide/transitions.md]: trigger for transition 'expend', like the exemple

2016-02-16 14:58 GMT-02:00 Evan You <notifications@github.com>:

Comment (yyx990803) [inline src/guide/transitions.md]: I mean, we should probably use `toggle` here so everyone can understand?

Comment (jhserodio) [inline src/guide/transitions.md]: for beginners, yes

2016-02-16 15:47 GMT-02:00 Evan You <notifications@github.com>:

Comment (azamat-sharapov) [inline src/guide/transitions.md]: Use **English** language, @jhserodio ! I don't understand what does **botao** mean. - This is what Evan means.

Comment (jhserodio) [inline src/guide/transitions.md]: sorry, botao = button, my fault

2016-02-17 4:22 GMT-02:00 Azamat <notifications@github.com>:

Comment (ludo237) [inline src/guide/transitions.md]: @jhserodio fix that please :+1:

**Comment by yyx990803** [comment]
Closing due to inactivity, feel free to re-submit if mentioned problems are fixed.

**Pull request closed without merging**